instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -473,6 +576,7 @@ def _get_system_disk_merged(
 
```

Comment:
Doesn't even look like this function is being used? Seems like the merged instrument is not supported yet. We should just remove this for the time being.

Reply:
I believe I have addressed this with the most recent commits